tidy3d/components/heat_charge/simulation.py

```diff
@@ -33,16 +35,55 @@
 
 from ...log import log
 
-HEAT_BACK_STRUCTURE_STR = "<<<HEAT_BACKGROUND_STRUCTURE>>>"
+HEAT_CHARGE_BACK_STRUCTURE_STR = "<<<HEAT_CHARGE_BACKGROUND_STRUCTURE>>>"
 
+HeatBCTypes = (TemperatureBC, HeatFluxBC, ConvectionBC)
+HeatSourceTypes = (UniformHeatSource, HeatSource, HeatFromElectricSource)
+ElectricBCTypes = (VoltageBC, CurrentBC, InsulatingBC)
 
-class HeatSimulation(AbstractSimulation):
-    """Contains all information about heat simulation.
+
+class HeatChargeSimulationType(str, Enum):
+    """Enumeration of the types of simulations currently supported"""
+
+    HEAT = "HEAT"
+    CONDUCTION = "CONDUCTION"
+
+
+class HeatChargeSimulation(AbstractSimulation):
+    """This class is used to define thermo-electric simulations.
+
+    Notes
+    -----
+        'HeatChargeSimulation' supports different types of simulations. It solves the
+        drift-diffusion equations (or simplified subsets of the same) using the
+        Finite-Volume (FV) method.
+
+        Currently, we support:
+            * Heat simulations: we solve the heat equation with specified heat sources,
+                BCs, etc.
+            * Conduction simulations: the conduction equation, div(sigma*grad(psi))=0,
+                (with sigma being the electric conductivity) is solved for specified BCs.
+
+        Coupling between these simulations is currently limited to 1-way coupling between
+        heat and conduction simulations. Coupling is specified by defining a heat source of
+        type 'HeatFromElectricSource'. With this coupling, joule heating is calculated as part
+        of the solution to a CONDUCTION simulation and then read in to the HEAT simulation.
+        When using coupling we anticipate two scenarios:
+            1. one in which BCs and sources are specified for both HEAT and CONDUCTION simulations.
+                In this case one mesh will be generated and used for both the CONDUCTION and HEAT
+                simulations.
+            2. only heat BCs/sources are provided. In this case, only the HEAT equation will be solved.
+                Before the simulation starts, it will try to load the heat source from file so a
+                previously run CONDUCTION simulations must have run previously. Since the CONDUCTION
```

Comment:
Just future improvement ideas: maybe we can make a little diagram to just illustrate this for people scrolling through the docs

Reply:
I like that idea! I'll try to prepare something, thought maybe for another PR?